egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -121,6 +129,10 @@ def CheckArgs(args):
         args.nonlin_input_dim = args.pnorm_input_dim
         args.nonlin_output_dim = args.pnorm_output_dim
 
+    if add_final_sigmoid and include_log_softmax:
```

Comment:
there is something called mutually_exclusive_group in argparse. Use that rather than manually performing the check.